test/controllers/patients_controller_test.rb

```diff
@@ -12,6 +15,65 @@ class PatientsControllerTest < ActionDispatch::IntegrationTest
     @clinic = create :clinic
   end
 
+  describe 'index method' do
+    before do
+      # Not sure if there is a better way to do this, but just calling
+      # `sign_in` a second time doesn't work as expected
+      delete destroy_user_session_path
```

Comment:
Woo!  I tried making a `sign_out` method to parallel `sign_in` in `test_helper.rb`, but there's an existing one that is Capybara-specific and is paired with `log_in` :)